### PR TITLE
fix: navbar i18n

### PR DESCRIPTION
**Pull request by juxianzheng**
Changes:

Some of the contents under "Resource" button in navigation bar of zh-cn page needs to be translated into Chinese.

Screenshots of the change:

Before

截屏2022-04-22 21 46 30

After

截屏2022-04-22 21 45 58

### CODE REVIEW

**Comment by yzeng25** [comment]
LGTM except some neats.

1. Changes in `yarn.lock` files should not be included. This file should remain the same. Please fix it.
2. Please modify your **pull request title** and **description** to something more precise and shorter in length. 

- Something like `chore: add translation for "Resource" button` for the pull request title; 
- Something like `Changes: In Chinese language, the "Resource" button in  navigation bar needs to be translated into - -Chinese` for description would be way better. 
- Delete the `Fixes: #[Add issue number here]` part,  since there is not an issue for it (You can definitely create an issue as well but it is up to you). 
- A couple of screenshots to show the before/after could help as well.

cc @Baoyuantop @guoqqqi @bzp2010 @juzhiyuan @SkyeYoung

File: website/i18n/zh/docusaurus-theme-classic/navbar.json [hunk 1]
```diff
@@ -11,8 +11,20 @@
...
+    "message": "插件中心",
+    "description": "Navbar item with label Plugin Hub"
+  },
+  "item.label.Community": {
+    "message": "社区活动",
+    "description": "Navbar item with label Community"
+  },
   "item.label.Events": {
-    "message": "Events",
+    "message": "活动介绍",
```
Comment (juzhiyuan) [inline website/i18n/zh/docusaurus-theme-classic/navbar.json]:
```suggestion
    "message": "社区活动",
```

File: website/i18n/zh/docusaurus-theme-classic/navbar.json [hunk 2]
```diff
@@ -11,8 +11,20 @@
...
+  "item.label.Resources": {
+    "message": "资源",
+    "description": "Navbar item with label Resources"
+  },
+  "item.label.Plugin Hub": {
+    "message": "插件中心",
+    "description": "Navbar item with label Plugin Hub"
+  },
+  "item.label.Community": {
+    "message": "社区活动",
```
Comment (juzhiyuan) [inline website/i18n/zh/docusaurus-theme-classic/navbar.json]: image

```suggestion
    "message": "加入社区",
```

File: website/i18n/zh/docusaurus-theme-classic/navbar.json [hunk 3]
```diff
@@ -11,8 +11,20 @@
     "message": "博客",
     "description": "Navbar item with label Blog"
   },
+  "item.label.Resources": {
+    "message": "资源",
+    "description": "Navbar item with label Resources"
+  },
+  "item.label.Plugin Hub": {
```
Comment (juzhiyuan) [inline website/i18n/zh/docusaurus-theme-classic/navbar.json]:
```suggestion
  "item.label.PluginHub": {
```

File: website/i18n/zh/docusaurus-theme-classic/navbar.json [hunk 4]
```diff
@@ -11,8 +11,20 @@
     "message": "博客",
     "description": "Navbar item with label Blog"
   },
+  "item.label.Resources": {
+    "message": "资源",
```
Comment (juzhiyuan) [inline website/i18n/zh/docusaurus-theme-classic/navbar.json]:
```suggestion
    "message": "相关资源",
```

**Comment by guoqqqi** [comment]
@juxianzheng 
You can see a preview of the changes in this section of the PR, please check it yourself after the changes are made
image

**Comment by guoqqqi** [comment]
Here are some other questions
image

**Comment by SkyeYoung** [comment]
> Here are some other questions image

@guoqqqi  `undefined` has been fixed in #1029.
@juxianzheng pls merge current master into the PR branch. If you encounter some problems, please feel free to ping me :D

**Comment by juzhiyuan** [comment]
> ### ✅ Deploy Preview for _apache-apisix_ ready!
> Name	Link
> 🔨 Latest commit	[882df08](https://github.com/apache/apisix-website/commit/882df08fa3276563f0ba293d268aeaaf754a60b1)
> 🔍 Latest deploy log	https://app.netlify.com/sites/apache-apisix/deploys/6257d6363f3bb00009631752
> 😎 Deploy Preview	https://deploy-preview-1024--apache-apisix.netlify.app
> 📱 Preview on mobile	
> Toggle QR Code...
> _To edit notification comments on pull requests, go to your [Netlify site settings](https://app.netlify.com/sites/apache-apisix/settings/deploys#deploy-notifications)._

Please note: each PR has its Preview Link. So you could see what has been changed.

image 

Plugin Hub needs to be updated with "插件市场"

---

@guoqqqi Maybe it's time to add more plugins to this page? As `apache/apisix` has supported more plugins.

**Review by juzhiyuan — DISMISSED** [review]

**Comment by Baoyuantop** [comment]
> > ### ✅ Deploy Preview for _apache-apisix_ ready!
> > Name	Link
> > 🔨 Latest commit	[882df08](https://github.com/apache/apisix-website/commit/882df08fa3276563f0ba293d268aeaaf754a60b1)
> > 🔍 Latest deploy log	https://app.netlify.com/sites/apache-apisix/deploys/6257d6363f3bb00009631752
> > 😎 Deploy Preview	https://deploy-preview-1024--apache-apisix.netlify.app
> > 📱 Preview on mobile
> > Toggle QR Code...
> > _To edit notification comments on pull requests, go to your [Netlify site settings](https://app.netlify.com/sites/apache-apisix/settings/deploys#deploy-notifications)._
> 
> Please note: each PR has its Preview Link. So you could see what has been changed.
> 
> image
> 
> Plugin Hub needs to be updated with "插件市场"

I noticed that this issue was not addressed🤔

File: website/i18n/zh/docusaurus-theme-classic/navbar.json [hunk 5]
```diff
@@ -11,8 +11,20 @@
     "message": "博客",
     "description": "Navbar item with label Blog"
   },
+  "item.label.Resources": {
+    "message": "相关资源",
+    "description": "Navbar item with label Resources"
+  },
+  "item.label.PluginHub": {
+    "message": "插件中心",
+    "description": "Navbar item with label Plugin Hub"
```
Comment (yzeng25) [inline website/i18n/zh/docusaurus-theme-classic/navbar.json]: I noticed @juxianzheng has translated it to Chinese, just like other lines. But as @Baoyuantop saw in preview, this part of change does not show up. Need to take a closer look @juzhiyuan @SkyeYoung .

Comment (SkyeYoung) [reply website/i18n/zh/docusaurus-theme-classic/navbar.json]: 🤔 I'll have a look tomorrow.

Comment (Baoyuantop) [reply website/i18n/zh/docusaurus-theme-classic/navbar.json]: And it should be "插件市场"

Comment (yzeng25) [reply website/i18n/zh/docusaurus-theme-classic/navbar.json]: > And it should be "插件市场"

Yeah but you need to get this part `"message": "插件中心",` to work first. The rest of it is just changing `"插件中心"` to `"插件市场"`.

Comment (SkyeYoung) [reply website/i18n/zh/docusaurus-theme-classic/navbar.json]: Here's a space. Emmm, I will change this label to "PluginHub". Wait a moment, I will create a new PR.

Comment (SkyeYoung) [reply website/i18n/zh/docusaurus-theme-classic/navbar.json]: It should be fixed in #1043

Comment (yzeng25) [reply website/i18n/zh/docusaurus-theme-classic/navbar.json]: > It should be fixed in #1043

Great :)

**Comment by SkyeYoung** [comment]
@juzhiyuan we need rerun the preview task to determine if the Plugin Hub's label problem has been fixed.

**Comment by SkyeYoung** [comment]
@Baoyuantop "插件市场" has been fixed.

**Review by guoqqqi — DISMISSED** [review]
LGTM

File: website/config/navbar.js [hunk 6]
```diff
@@ -72,7 +72,7 @@ module.exports = [
       },
       {
         to: '/plugins',
-        label: 'Plugin Hub',
+        label: 'PluginHub',
```
Comment (SkyeYoung) [inline website/config/navbar.js]: Why you modified this file but not merge master into this branch?

**Review by SkyeYoung — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
```
Comment (guoqqqi) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Please check the files you have changed before submitting, Don't submit unrelated changes

**Review by guoqqqi — CHANGES_REQUESTED** [review]

**Review by guoqqqi — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]
LGTM. I fixed the pull request description as well. Please try to fix them next time :)

Thanks @juxianzheng for your contribution, really appreciate it!

**Pull request merged**